clase 1 19.4/src/main/java/scanner/impl/MenuSizeImpl.java

```diff
@@ -0,0 +1,19 @@
+package scanner.impl;
+
+import scanner.ScannerClass;
+import java.util.Set;
+import java.util.TreeSet;
+
+public class MenuSizeImpl extends ScannerClass {
```

Comment:
I do not think you need to extend from Scanner, I think if you want to apply inheritance here, you can have a ScannerClass to ask for game mode (askUserForWayOfPlaying), and another class for ask board size (askUserForBoardSize), but I think in essence they are Scanners, so, you can have only one class to ask for those information